composer.json

```
@@ -15,11 +15,11 @@
 	},
 	"require": {
 		"masterminds/html5": "^2.8",
-		"symfony/dom-crawler": "^5.4",
-		"symfony/css-selector": "^5.4"
+		"symfony/dom-crawler": "^6.0",
+		"symfony/css-selector": "^6.0"
 	},
 	"require-dev": {
-		"php": ">=7.4",
+		"php": ">=8.0",
```

```suggestion
		"php": ">=8.1",
```
DOM Crawler requires at least 8.1, so it could make sense to bump on our end as well.